custom_components/smartthinq_sensors/wideq/device.py

```diff
@@ -567,6 +567,8 @@ def _get_current_temp_key(key: str, data):
     def decode_snapshot(self, data, key):
         """Decode  status data."""
         decoded = {}
+        if self._data["Monitoring"]["type"] == "JSON":
+            return data
```

Comment:
Can you share `json info` for your device? I would understand if can be brought back to standard decoding.

Reply:
This is not need if use solution in another conversation.

Reply:
Ok, so just remove this.